Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,34 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do not register it)
+ 
+# Stop services
+net stop "Print Connector service"; net stop PrintConnectorUpdaterSvc
+ 
+# Restore local printers
+c:\windows\system32\spool\tools\PrintBrm.exe -R -F C:\ConnectorBackup\printers.brm
```

Comment:
what if C isn't thier root drive?

Reply:
Given that this are sample scripts I think it is fine to have them modify it to their specific needs. But we can wait to see if Phil has an opinion on that

Reply:
OK. Let's note down what to keep in the public documentation: changing root drive and maybe disabling BRM if needed